### PR TITLE
Shawn beasley otrs intros

### PR DESCRIPTION
**Pull request by ShawnBeasleyOTRS**
This update contains introduction to the following chapters.

Communication & Notifications
https://doc.otrs.com/doc/manual/admin/stable/en/content/communication-notifications.html

External Interface
https://doc.otrs.com/doc/manual/admin/stable/en/content/external-interface.html

OTRS Group Services
https://doc.otrs.com/doc/manual/admin/stable/en/content/otrs-group-services.html

Users, Groups & Roles
https://doc.otrs.com/doc/manual/admin/stable/en/content/users-groups-roles.html

### CODE REVIEW

File: content/communication-notifications.rst [hunk 1]
```diff
@@ -1,7 +1,11 @@
 Communication & Notifications
 =============================
 
-This is the chapter *Communication & Notifications*.
+Clear, complete, and transparent communication is key to ensuring that your organization offers outstanding service to your customers. You organization would like to give your customers the most consistent service and ensure that your service is highly recognizable, every time. Your teams want to have an identity which instills a specific sense of security and familiarity.
+
+OTRS provides you with the tools to build teams, trust and security to better an uniformly service your customer. With system addresses you can assign your inbound mails to certain teams and allow them all to use the same address without causing confusion by using team mailboxes via a mail client. Templates, automatic answers, and attachment management let you leverage central management of the most important communacitons. Many other tools are also available and covered in the sections below.
```
Comment (urbalazs) [inline content/communication-notifications.rst]: I think, a verb is missing here: to better an uniformly service

File: content/external-interface.rst [hunk 2]
```diff
@@ -1,7 +1,12 @@
 External Interface
 ==================
 
-This is the chapter *External Interface*.
+The need to transport valuable information about service to users is a major one for a service desk. Not always is it possible or plausible to do this via a corporate internet. The customers of a service desk need a one-stop solution for all their service needs.
```
Comment (urbalazs) [inline content/external-interface.rst]: Can you explain me, what is "one"?
to users is a major one...

File: content/users-groups-roles.rst [hunk 3]
```diff
@@ -1,7 +1,9 @@
 Users, Groups & Roles
 =====================
 
-This is the chapter *Users, Groups & Roles*.
+Simple and complex organizations need a way to flexibily control access to their service desk. Access, resources, and permissions must be orchestrated so that, the users and customers have access to the resources they need and their data is protected as needed by the system.
```
Comment (urbalazs) [inline content/users-groups-roles.rst]: What does it mean "flexibily"?

**Comment by urbalazs** [comment]
Hello @ShawnBeasleyOTRS 
Thank you for the new content. I added some inline comments to your suggestions. There are also some spelling errors in your text, please fix them.

File: content/users-groups-roles.rst (hunk 3 above)
Comment (ShawnBeasleyOTRS) [reply content/users-groups-roles.rst]: The need is a major one.

File: content/external-interface.rst (hunk 2 above)
Comment (ShawnBeasleyOTRS) [reply content/external-interface.rst]: The need is a major one.

**Comment by urbalazs** [comment]
Thank you for the contribution, it was merge with https://github.com/OTRS/doc-admin/commit/29f29a9f84b026938588ecc9fe551085ea5dd4fa with slightly changes.

**Pull request closed without merging**